FinSearchUnified/Tests/Subscriber/WidgetsTest.php

```diff
@@ -19,9 +24,8 @@ public static function setUpBeforeClass()
 
     protected function tearDown()
     {
+        unset(Shopware()->Session()->isSearchPage, Shopware()->Session()->isCategoryPage);
         parent::tearDown();
-
-        Shopware()->Session()->offsetUnset('isSearchPage');
```

Comment:
Unfortunately `offsetUnset` does not return `$this`, so you can not chain those statements. How sad.